raytracing/imagingpath.py

```diff
@@ -423,50 +433,62 @@ def save(self, filepath,
 
     def drawObject(self, axes):  # pragma: no cover
         """ Draw the object as defined by objectPosition, objectHeight """
-        (xScaling, yScaling) = self.axesToDataScaling(axes)
-        arrowWidth = xScaling * 0.01
-        arrowHeight = yScaling * 0.03
+        (xScaling, yScaling) = self.axesScale(axes)
+        arrowHeadHeight = self.objectHeight * 0.1
+
+        heightFactor = self.objectHeight / yScaling
+        arrowHeadWidth = xScaling * 0.01 * (heightFactor/0.2) ** (3/4)
 
         axes.arrow(
             self.objectPosition,
             -self.objectHeight / 2,
             0,
             self.objectHeight,
-            width=arrowWidth / 5,
+            width=arrowHeadWidth / 5,
             fc='b',
             ec='b',
-            head_length=arrowHeight,
-            head_width=arrowWidth,
+            head_length=arrowHeadHeight,
+            head_width=arrowHeadWidth,
             length_includes_head=True)
 
-    def drawImages(self, axes):  # pragma: no cover
-        """ Draw all images (real and virtual) of the object defined by 
-        objectPosition, objectHeight """
-
-        (xScaling, yScaling) = self.axesToDataScaling(axes)
-        arrowWidth = xScaling * 0.01
-        arrowHeight = yScaling * 0.03
-
+    def computeImages(self):
```

Comment:
Let's rename this `intermediateForwardConjugates(self)` and return the array instead of setting the internal variable. Let's also drop the internal variable (`self.images`).  We'll just compute it every time because setting an internal variable on a computation "in the background" is not the best practice: what if someone tries to use images without calling `computeImages()`?  Since the computation is not expensive at all (not much is intensive in this module), let's recompute it every time.